framework/source/class/qx/util/ResourceManager.js

```diff
@@ -291,8 +298,16 @@ qx.Class.define("qx.util.ResourceManager",
           }
           // If the resourceUri begins with a single slash, include the current
           // hostname
-          else if (resourceUri.match(/^\//) != null) {
-            statics.__urlPrefix[lib] = window.location.protocol + "//" + window.location.host;
+          else if (resourceUri.match(/^\//) != null)
```

Comment:
For now I'm just looking at the existing code. It seems to differ when using curly braces for function bodies, objects, control structures. There's no style guide I can find in the source, though. It is just mentioned for the tech docs. It would be nice to have something specific, because running the *fix* generator job might just create hundreds of changed lines.